eslint.config.mjs

```diff
@@ -1,6 +1,62 @@
 import baseConfig from '@hono/eslint-config'
 import { defineConfig, globalIgnores } from 'eslint/config'
 
-export default defineConfig(globalIgnores(['.yarn', '**/dist']), {
+export default defineConfig(globalIgnores(['.yarn', '**/coverage', '**/dist']), {
   extends: baseConfig,
+
+  languageOptions: {
+    parserOptions: {
+      projectService: true,
+      tsconfigRootDir: import.meta.dirname,
+    },
+  },
```

Comment:
This tells the `typescript-eslint` parser where to find the root `tsconfig.json`, and to use project references